tach/cli.py

```diff
@@ -207,15 +207,23 @@ def parse_arguments(
 ) -> tuple[argparse.Namespace, argparse.ArgumentParser]:
     parser = build_parser()
     parsed_args = parser.parse_args(args)
+
+    if len(args) < 1 or args[0] not in ["pkg", "clean", "sync"]:
+        # TODO: unify project config handling
+        fs.validate_project_config_path()
+
```

Comment:
This got removed entirely in main, each command is already validating project config if necessary, and this code actually causes failures when running `tach check --root [otherdir]`